src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -622,6 +621,7 @@ AppendShardIdToName(char **name, uint64 shardId)
 	{
 		snprintf(extendedName, NAMEDATALEN, "%s%s", (*name), shardIdAndSeparator);
 	}
+
```

Comment:
I think citus_indent put this new line. I am not sure why it is not added before (probably it is related with my version of citus_indent) but it kinda makes sense.